scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -471,5 +463,14 @@ function verify_kubernetes_version_upgrade() {
             break
         fi
     done
+}
+
+function point_to_target_cluster() {
+    kubectl get secrets "${TARGET_KUBECONFIG_SECRET}" -n "${NAMESPACE}" -o json | jq -r '.data.value' | base64 -d >"${TARGET_KUBECONFIG_FILE}"
+    export KUBECONFIG="${TARGET_KUBECONFIG_FILE}"
+
+}
 
+function point_to_ephemeral_cluster() {
```

Comment:
My experience is that 'management' cluster is more commonly used

Reply:
changed.